rust_backend/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "rust_backend"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+anyhow = "1.0.82"
+console_error_panic_hook = "0.1.6"
+console_log = "1.0.0"
+geo = "0.28.0"
+geojson = { version = "0.24.1", features = ["geo-types"] }
+js-sys = "0.3.69"
+log = "0.4.20"
+serde = "1.0.188"
+serde_json = "1.0.105"
+serde-wasm-bindgen = "0.6.0"
+wasm-bindgen = "0.2.87"
+web-sys = { version = "0.3.64", features = ["console"] }
+web-time = "1.1.0"
+
+# For local development, build dependencies in release mode once, but otherwise
```

Comment:
This is _key_. Dev builds are subsecond (and even when things grow, very fast). Release builds are slow,but only need to happen on GH actions